Slavahaim/hometask-6/spec/my_hash_spec.rb

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require '../my_hash'
+require 'rspec'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe MyHash do
+  let(:my_hash) { described_class.new }
+
+  describe '#[]=' do
+    before do
+      my_hash[1] = 1
+      my_hash[2] = 2
+      my_hash['s'] = 's'
+    end
+
+    it 'return changed hash size and write key=>value' do
```

Comment:
We can split this `it` into 2 `it`'s
- `returns changed hash size`
- `writes key`